make.sh

```diff
@@ -136,6 +157,6 @@ case "$1" in
   "linux32" ) build_linux32;;
   "msvc_update_genfiles" ) msvc_update_genfiles;;
   * )
-    echo "Usage: $0 ["`grep '^  "' $0 | cut -d '"' -f 2 | tr "\\n" "|"`"]"
```

Comment:
the old solution is nice because it is quite future proof, so i feel that we better keep that approach?

Reply:
I think keeping description for each build option in `usage` is quite easy.
If this PR is merged, everyone would take a look in `usage` function first, then 
edit `usage` description appropriate.

Reply:
agreed, let see if there is any other comments on Shell coding.